app/components/article-content.js

```diff
@@ -104,6 +107,31 @@ export default Component.extend(
 					label
 				});
 			}
+
+			if (figure) {
+				let lightboxModel;
+
+				if (gallery) {
+					try {
+						lightboxModel = JSON.parse(gallery.getAttribute('data-attrs'));
```

Comment:
sorry to be picky but
this looks almost exactly like code in 'else'
maybe we could extract it to a method?
getLightboxModel(elem); -> object or something like that?

Reply:
👍